pom.xml

```diff
@@ -188,7 +211,7 @@
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
-      <scope>runtime</scope>
```

Comment:
please keept the H2 DB for all Developers that cant run docker on their systems

Reply:
I changed because H2 should be only for testing, not runtime, and H2 database should not be loaded in Production environment.